modules/API/pages/built-in-endpoints.adoc

```diff
@@ -2664,18 +2293,68 @@ Response::
 |===
 | Name | Required | Description
 
+| `timeout`
+| No
+| Integer that specifies the number of seconds after which the query will time out. If the parameter is set to 0 or isn't provided, the system-wide endpoint timeout setting is applied.
+|===
+
+=== Delete an edge by source, target, edge type and discriminator
+
+[source.wrap,console]
+----
+DELETE /graph/{graph_name}/edges/{source_vertex_type}/{source_vertex_id}/{edge_type}
+/{target_vertex_type}/{target_vertex_id}/{discriminator}
+----
+
+This endpoint allows you to delete an edge by its source, target, edge type, and
+xref:gsql-ref:ddl-and-loading:defining-a-graph-schema.adoc#_discriminator[discriminator].
+
+This endpoint requires the `DELETE_DATA` privilege on the type of the edge being queried.
+
+==== Sample request
+
+[tabs]
+====
+Single-attribute discriminator::
++
+--
+[.wrap,console]
+----
+curl -X DELETE 'http://localhost:9000/graph/multiedge/edges/Person/4/transfer/Person/30/2431'
+----
+--
+Composite discriminator::
++
+--
+[.wrap,console]
+----
+curl -X DELETE "http://localhost:9000/graph/multiedge/edges/Person/9/transfer2/Account/9/77,1117,2010-04-09%2003:57:01" <1>
```

Comment:
The DateTime discriminator attribute can be a string format(string eg 2010-04-09%2003:57:01) or an epoach time(unit, eg 1666892012)